client/components/purchase-detail/index.jsx

```diff
@@ -7,18 +7,22 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import Button from 'components/button';
 import Gridicon from 'components/gridicon';
+import PurchaseButton from './purchase-button';
+import TipInfo from './tip-info';
 
 const PurchaseDetail = ( {
+	body,
 	buttonText,
 	description,
 	href,
 	icon,
+	id,
+	info,
 	isPlaceholder,
 	isRequired,
 	isSubmitting,
-	onClick,
+	onClick = () => {},
```

Comment:
Actually I'm not sure about pros and cons to use it in this way. I guess is simplest.